modules/openapi-generator/src/main/resources/python/python-experimental/api_client.mustache

```diff
@@ -542,3 +569,179 @@ class ApiClient(object):
+        now = datetime.datetime.now()
+        stamp = mktime(now.timetuple())
+        cdate = formatdate(timeval=stamp, localtime=False, usegmt=True)
+        created = now.strftime("%s")
+
+        signed_headers_dict = {}
+        request_headers_dict = {}
+        for hdr_key in self.configuration.signed_headers_dict:
+            hdr_key = hdr_key.lower()
+            if hdr_key == '(request-target)':
+                value = request_target
+            elif hdr_key == '(created)':
+                value = created
+            elif hdr_key == 'date':
+                value = cdate
+                request_headers_dict['Date'] = '{0}'.format(cdate)
+            elif hdr_key == 'digest':
+                request_body = body.encode()
+                body_digest, digest_prefix = self.get_message_digest(request_body)
+                b64_body_digest = b64encode(body_digest.digest())
+                value = digest_prefix + b64_body_digest.decode('ascii')
+                request_headers_dict['Digest'] = '{0}{1}'.format(digest_prefix, b64_body_digest.decode('ascii'))
+            elif hdr_key == 'host':
+                value = target_host
+                request_headers_dict['Host'] = '{0}'.format(target_host)
+            else:
+                value = headers[hdr_key]
+            signed_headers_dict[hdr_key] = value
+
+        # If the user has not provided any signed_headers, the default must be set to '(created)'.
+        if len(self.configuration.signed_headers_dict) == 0:
+            signed_headers_dict['(created)'] = created
+
+        return signed_header_dict, request_headers_dict
+
+    def get_http_signature_headers(self, resource_path, method, headers, body, query_params):
+        """
+        Create a cryptographic message signature for the HTTP request and add the signed headers.
+        
+        :param resource_path : A string representation of the HTTP request resource path.
+        :param method: A string representation of the HTTP request method, e.g. GET, POST.
+        :param headers: A dict containing the HTTP request headers.
+        :param body: The string representation of the HTTP request body.
+        :param query_params: A string representing the HTTP request query parameters.
+        :return: A dict of HTTP headers that must be added to the outbound HTTP request.
+        """
+        if method is None:
+            raise Exception("HTTP method must be set")
+        if resource_path is None:
+            raise Exception("Resource path must be set")
+
+        signed_headers_dict, request_headers_dict = self.get_signed_header_info(self,
+            resource_path, method, headers, body, query_params)
+
+        string_to_sign = ""
+        for i, (key, value) in enumerate(signed_headers_dict.items()):
+            string_to_sign += "{0}: {1}".format(key.lower(), value)
+            if i > 0:
+                string_to_sign += "\n"
```

Comment:
How about changing this to something like?
```
header_items = ["{0}: {1}".format(key.lower(), value) for key, value in signed_headers_dict.items()]
string_to_sign = "\n".join(header_items)
```
That saves us 3 lines

Reply:
done